crates/re_types/definitions/rerun/archetypes/mesh3d.fbs

```diff
@@ -22,19 +22,15 @@ table Mesh3D (
 
   /// The positions of each vertex.
   ///
-  /// If no `indices` are specified, then each triplet of positions is interpreted as a triangle.
+  /// If no `triangle_indices` are specified, then each triplet of positions is interpreted as a triangle.
   vertex_positions: [rerun.components.Position3D] ("attr.rerun.component_required", order: 1000);
 
   // --- Recommended ---
 
-  /// Optional properties for the mesh as a whole (including indexed drawing).
-  //
-  // NOTE: We cannot have triangle indices here as that would break our instance key rules (either 0, 1 or N).
-  mesh_properties: rerun.components.MeshProperties ("attr.rerun.component_recommended", nullable, order: 2000);
+  /// Optional indices for the triangles that make up the mesh.
+  triangle_indices: [rerun.components.UVector3D] ("attr.rerun.component_recommended", nullable, order: 2000);
```

Comment:
I think it makes more sense to call this component `TriangleIndices` instead of `UVector3D`.
When you see `UVector3D` in the component list in the viewer you may be confused thinking it is the positions or normals or something.